h/schemas/base.py

```diff
@@ -99,6 +100,61 @@ def serialize(self, node, appstruct):
     return EnumType
 
 
+def _colander_exception_msg(exc):
+    """
+    Combine error messages from a `colander.Invalid` exception.
```

Comment:
Docstring/rST/Sphinx is tricksy. I found that the following format worked well for this docstring:

```
Combine error messages from a :py:class:`colander.Invalid` exception.

:type exc: :py:class:`colander.Invalid`
:rtype: str
```

Reply:
To summarize a Slack discussion - the `:py:class:` annotation is not needed here because Sphinx is already aware that the value should be a type in this context and `:type exc: ValidationError` will generate a working cross-link. Only in the main docstring body is `:class:Foo` needed.